examples/chip-tool/templates/commands.zapt

```diff
@@ -17,25 +17,24 @@
 {{#chip_client_clusters}}
 {{> cluster_header}}
 
-{{#chip_cluster_commands}}
+{{#zcl_commands_source_client}}
 /*
  * Command {{asUpperCamelCase name}}
  */
-class {{asUpperCamelCase clusterName}}{{asUpperCamelCase name}}: public ClusterCommand
```

Comment:
The fact that different helpers in ZAP provide different state is ... annoying.  Please file an issue on them?  Because I think other helpers they have _do_ provide clusterName......

Reply:
https://github.com/project-chip/zap/issues/408